audio-worklet/src/bitcrusher-script-processor.js

```diff
@@ -50,12 +50,8 @@ class Bitcrusher {
 
     // Index and previousSample defined as globals to handle block transitions.
     // There is one of each for every channel to handle multi-channel input.
-    this.indexes_ = new Array();
-    this.previousSamples_ = new Array();
-    for (let i = 0; i < channels; i++) {
-      this.indexes_.push(0);
-      this.previousSamples_.push(0);
-    }
+    this.indexes_ = new Array(channels).fill(0);
```

Comment:
The meaning of indexes probably isn't clear since this change would break it. It's supposed to hold the current index (which is used to compute when to sample reduce) for each channel. So the actual index (the contents of the array) could be much larger than what you could represent with 8 bits.